submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/SubmissionRequestResource.kt

```diff
@@ -23,4 +24,13 @@ class SubmissionRequestResource(
     ): SubmissionRequest {
         return submissionRequestService.getSubmissionRequest(accNo, version)
     }
+
+    @GetMapping("/{accNo}/{version}/status")
+    @ResponseBody
+    suspend fun getSubmissionRequestStatus(
```

Comment:
not part of this ticket but given we are adding this endpoint, why not publish everything? it can be super usefull for others matters like debugging purposes.

Reply:
There's already an endpoint that exposes everything. I just added one specifically for the status